### DISCUSSION TITLE
Error using versioned API route

### DISCUSSION THREAD

**Discussion by jinsley8** [nextauthjs/next-auth#6509] [Help] [answered]
I'm using a versioned API route like `/api/v1/auth` but I'm getting errors that `next-auth` cannot find the `/api/auth` routes it needs.

I'm getting `GET: 404 - http://localhost:3000/api/auth/session` and `POST: 404 - http://localhost:3000/api/auth/_log`

How can I set it up to work with my versioned API?

`[...nextauth].ts` is at the following location: `/api/v1/auth/[...nextauth].ts`

**Comment by kn327** [comment]
Your file needs to be named `[...nextauth].ts`, not `[...next-auth].ts` (with an extra `-`)

**Reply by jinsley8** [reply]
Typo in my post

**Comment by jinsley8** [comment, marked as answer]
Figured it out, answer is here for anyone looking:

https://next-auth.js.org/getting-started/client#custom-base-path
